grammar.js

```diff
@@ -1366,8 +1369,7 @@ module.exports = grammar({
      * PostfixExpr [Ascription]
      */
     ascription_expression: $ =>
-      prec.dynamic(
```

Comment:
Lack of fail-fast `left` precedence with conjunction of introduced `applied_constructor_type` could treat a body of the this class
```scala
class B {
  self: Something[A] => 

  case class Hello(a: Int)
}
```
as a giant `ascription_exception` with `Something[A] => \n case class Hello[a]` being a type signature.
<img width="1962" height="1902" alt="image" src="https://github.com/user-attachments/assets/9f3daeaa-29bc-483b-ad94-1f58c7fb5798" />